win32ss/user/ntuser/simplecall.c

```diff
@@ -706,7 +706,53 @@ NtUserCallHwndOpt(
     return hWnd;
 }
 
-DWORD
+static HWND
+IntGetEnabledPopup(HWND hWnd)
+{
+    PTHREADINFO pti;
+    BOOL bFlag = FALSE;
+    PWND pwndNode, pwndOwner, pWnd = UserGetWindowObject(hWnd);
+
+    if (!pWnd)
+        return NULL;
+
+    pti = pWnd->head.pti;
+    pwndNode = pWnd->spwndNext;
+
+    while (pwndNode != pWnd)
```

Comment:
Couldn't this be rewritten with a for-loop instead?

Reply:
Implementation is deleted.